VMEncryption/main/oscrypto/OSEncryptionState.py

```diff
@@ -54,7 +54,8 @@ def __init__(self, state_name, context):
                                                             disk_util=self.disk_util)
 
         self.bek_util = BekUtil(disk_util=self.disk_util,
-                                logger=self.context.logger)
+                                logger=self.context.logger,
+                                encryption_environment=self.context.encryption_environment)
```

Comment:
nit: Probably encryption_env parameter could have been made optional to avoid any change in OS encryption path. No need to make the change now but something to consider for future changes.